pymc3/parallel_sampling.py

```diff
@@ -347,9 +347,7 @@ def __init__(
         progressbar=True,
     ):
         if progressbar:
-            import tqdm
-
-            tqdm_ = tqdm.tqdm
+            from tqdm.auto import tqdm as tqdm_
```

Comment:
We can probably just change `tqdm_` below to `tqdm`, the existing pattern here is pretty weird.